main.go

```diff
@@ -45,4 +55,15 @@ func main() {
 	close(swapIntentOperationBroadcastChan)
 	close(solverOperationSubmitChan)
 	close(governanceSignatureChan)
+
+	// Questions:
+	// entities:
+	//    backend - is this the bundler that gets the intent, distribute to solvers, collect solutio and create the metaTransactions? you can it backend and not bundler as there can be other configurations?
+	//    governance - what is this entity? is it part of the dApp or part of Atlas?
```

Comment:
The governance is part of the dApp, it's the entity authorized to sign the dApp operation (required parameter for Atlas.metacall)

Reply:
got it. Not sure I see the use case why the backend and governance is not the same but I guess it can happen.